tensorflow/lite/micro/micro_allocator.h

```diff
@@ -248,8 +248,9 @@ class MicroAllocator {
   virtual TfLiteStatus AllocateTfLiteEvalTensors(
       const Model* model, SubgraphAllocations* subgraph_allocations);
   // Allocates persistent tensor buffers for variable tensors in the subgraph.
-  virtual TfLiteStatus AllocateVariables(const SubGraph* subgraph,
-                                         TfLiteEvalTensor* eval_tensors);
+  virtual TfLiteStatus AllocateVariables(
```

Comment:
can you add a comment about what the offline_planner_offsets parameter is for?

Reply:
done